packages/workflows/src/ios.workflows.ts

```diff
@@ -19,6 +19,20 @@ const childWorkflows: IDeploymentWorkflows = {
       },
     ],
   },
+  set_icons_and_splash_images: {
+    label:
+      "Generate app launcher icon and splash screen image from icon.png and splash.png asset respectively. Then if provided, it will generate adaptive icon from icon-foreground.png and icon-background.png. Copy generated files to relevant folders.",
+    steps: [
+      {
+        name: "set_icons_and_splash_images",
+        function: async ({ tasks, config }) =>
+          tasks.ios.set_icons_and_splash_images({ assetPath: config.ios.assets_path }),
+      },
+    ],
+  },
+  // Generate iOS assets from source images (splash.png, icon.png and, optionally, icon-foreground.png and icon-background.png)
```

Comment:
I think `icon.png` should be `icon-only.png` instead

Reply:
I am not too sure what this comment is specifically referencing. When I checked my assets, the image was called `icon.png`. However, I think any name changes can be solved by accessing the Google Drive.

Reply:
I was going off the [docs](https://capacitorjs.com/docs/guides/splash-screens-and-icons), which suggest that the source asset should be called "icon-only.png". So I think that's what this comment should specify as the name for the source file too.

Reply:
Okay will do